packages/flutter/test/widgets/slivers_evil_test.dart

```diff
@@ -0,0 +1,185 @@
+            new SliverAppBar(delegate: new TestSliverAppBarDelegate(250.0), pinned: true),
+            new SliverToBoxAdapter(child: new Container(height: 5.0)),
+            new SliverAppBar(delegate: new TestSliverAppBarDelegate(250.0), pinned: true),
+            new SliverToBoxAdapter(child: new Container(height: 5.0)),
+            new SliverAppBar(delegate: new TestSliverAppBarDelegate(250.0), pinned: true),
+                        new SliverAppBar(delegate: new TestSliverAppBarDelegate(250.0), pinned: true),
+            new SliverToBoxAdapter(child: new Container(height: 5.0)),
+            new SliverAppBar(delegate: new TestSliverAppBarDelegate(250.0), pinned: true),
+            new SliverAppBar(delegate: new TestSliverAppBarDelegate(250.0)),
+            new SliverToBoxAdapter(child: new Container(height: 520.0)),
+            new SliverAppBar(delegate: new TestSliverAppBarDelegate(150.0), floating: true),
+            new SliverToBoxAdapter(child: new Container(height: 520.0)),
+            new SliverAppBar(delegate: new TestSliverAppBarDelegate(150.0), floating: true),
+            new SliverToBoxAdapter(child: new Container(height: 5.0)),
+            new SliverBlock(
+              delegate: new SliverBlockChildListDelegate(<Widget>[
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+                new Container(height: 50.0),
+              ]),
+            ),
+            new SliverAppBar(delegate: new TestSliverAppBarDelegate(250.0)),
+            new SliverAppBar(delegate: new TestSliverAppBarDelegate(250.0)),
+            new SliverAppBar(delegate: new TestSliverAppBarDelegate(250.0)),
+            new SliverPadding(
+              padding: new EdgeInsets.symmetric(horizontal: 50.0),
+              child: new SliverToBoxAdapter(child: new Container(height: 520.0)),
+            ),
+            new SliverToBoxAdapter(child: new Container(height: 520.0)),
+            new SliverToBoxAdapter(child: new Container(height: 520.0)),
+            new SliverToBoxAdapter(child: new Container(height: 5.0)),
+          ],
+        ),
+      ),
+    );
+    AbsoluteScrollPosition position = scrollableKey.currentState.position;
+    position.animate(to: 10000.0, curve: Curves.linear, duration: const Duration(minutes: 1));
+    await tester.pumpUntilNoTransientCallbacks(const Duration(milliseconds: 10));
```

Comment:
Can we change the combination of 1 minute and 10 millisecond intervals?  That seems like it's going to be pretty slow...